src/Uno.Toolkit.UI/Controls/LoadingView/LoadingView.HotReload.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection.Metadata;
+using System.Threading.Tasks;
+#if IS_WINUI
+using Microsoft.UI.Xaml;
+#else
+using Windows.UI.Xaml;
+#endif
+using Uno.Toolkit.UI;
+
+[assembly: ElementMetadataUpdateHandlerAttribute(typeof(LoadingView), typeof(LoadingViewElementMetadataUpdateHandler))]
+[assembly: ElementMetadataUpdateHandlerAttribute(typeof(ExtendedSplashScreen), typeof(LoadingViewElementMetadataUpdateHandler))]
```

Comment:
do we need this for subclass? ESS is already a LV

Reply:
+1